src/PhpSpreadsheet/Writer/Xlsx/Drawing.php

```diff
@@ -142,10 +147,11 @@ public function writeChart(XMLWriter $objWriter, \PhpOffice\PhpSpreadsheet\Chart
      * @param XMLWriter $objWriter XML Writer
      * @param BaseDrawing $pDrawing
      * @param int $pRelationId
+     * @param int $pHlinkClickId
      *
      * @throws WriterException
      */
-    public function writeDrawing(XMLWriter $objWriter, BaseDrawing $pDrawing, $pRelationId = -1)
+    public function writeDrawing(XMLWriter $objWriter, BaseDrawing $pDrawing, $pRelationId = -1, $pHlinkClickId = -1)
```

Comment:
Let's use `null` instead of `-1` for a better semantic of absence of value. Then we should also change `@param null|int $pHlinkClickId`.